module/Product/config/module.config.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+namespace Product;
+
+use Zend\Router\Http\Literal;
+use Zend\Router\Http\Segment;
+use Zend\ServiceManager\Factory\InvokableFactory;
+
+return [
+    'router' => [
+      'routes' => [
+          'product-list' => [
+              'type'    => Segment::class,
+              'options' => [
+                  'route'    => '/product/list[/:id]',
+                  'defaults' => [
+                      'controller' => Controller\ProductController::class,
+                      'action'     => 'index',
+                  ],
+              ],
+          ],
+          'product-add' => [
+              'type'    => Segment::class,
+              'options' => [
+                  'route'    => '/product/add',
+                  'defaults' => [
+                      'controller' => Controller\ProductController::class,
+                      'action'     => 'add',
+                  ],
+              ],
+          ],
+          'product-edit' => [
+              'type'    => Segment::class,
+              'options' => [
+                  'route'    => '/product/edit[/:id]',
+                  'defaults' => [
+                      'controller' => Controller\ProductController::class,
+                      'action'     => 'edit',
+                  ],
+              ],
+          ],
+          'product-add-ajax' => [
+              'type'    => Segment::class,
+              'options' => [
+                  'route'    => '/product/post',
```

Comment:
To keep things simple/obvious, it's usually best to name routes and actions the same way (i.e. either use `addAjax` for both the route name and the action -or- us `post` for both the route name and the action, not a mix of both).

Sometimes we'll need to break this rule for SEO or other reasons, but this isn't one of those times.